plexpy/datafactory.py

```diff
@@ -1156,6 +1156,19 @@ def get_watch_time_stats(self, rating_key=None, grouping=None, query_days=None):
 
         group_by = 'session_history.reference_id' if grouping else 'session_history.id'
 
+        _rating_keys = []
+        if media_type == 'collection':
+            pms_connect = pmsconnect.PmsConnect()
+            result = pms_connect.get_item_children(rating_key=rating_key)
+
+            for child in result['children_list']:
+                _rating_keys.append(child['rating_key'])
+        else:
+            _rating_keys.append(rating_key)
+
+        rating_keys = ','.join(_rating_keys)
+        rating_keys_arg = ','.join(['?'] * len(rating_keys))
```

Comment:
Forgot the underscore.
```suggestion
        rating_keys_arg = ','.join(['?'] * len(_rating_keys))
```

Reply:
Yeah forgot to test it on my side.